apps/database/buildspec.yml

```
@@ -0,0 +1,14 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.9
+    commands:
+      - apt update
+      - apt install -y postgresql-12
+      - make install-pypgstac
+      - pyenv rehash
+  build:
+    commands:
+      - make configure-database db_host=$PGHOST db_admin_password=$PGPASSWORD db_read_password=$READ_PASSWORD
```

We'll want to do this for all `make` commands that could echo passwords (not sure if this is the only one, so let's do a repo-wide search for `make` before we merge).
```suggestion
      - make -s configure-database db_host=$PGHOST db_admin_password=$PGPASSWORD db_read_password=$READ_PASSWORD
```

---

Looks like this is the only place where we need that flag.

---

Not needed, because CodeBuild masks those values in the log because they're from Secrest Manager.